content/en/platform/corda/4.10/community/api-testing.md

```diff
@@ -257,8 +257,8 @@ attributes.
 ### Further examples
 
 
-* See the flow testing tutorial [here](../../../../tutorials/corda/4.10/community/build-basic-cordapp/basic-cordapp-unit-testing.md)
-* See the oracle tutorial [here](../../../../tutorials/corda/4.10/community/supplementary-tutorials/oracles.md) for information on testing `@CordaService` classes
+* See the flow testing tutorial [here](../enterprise/get-started/tutorials/build-basic-cordapp/basic-cordapp-unit-testing.md)
```

Comment:
We're in 4.10 community edition file - why are we referencing the enterprise links ?
Check other occurrences .